app-backend/db/src/main/scala/com/azavea/rf/database/meta/PermissionsMeta.scala

```diff
@@ -0,0 +1,19 @@
+package com.azavea.rf.database.meta
+
+import com.azavea.rf.datamodel._
```

Comment:
Are all of these imports necessary?

Reply:
Good point. I cleaned them up in the latest commit.

Reply:
Good point. I cleaned them up in the latest commit.